Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -116,14 +117,33 @@ private void BladeOnVisibilityChanged(object sender, Visibility visibility)
                 BladeOpened?.Invoke(this, blade);
                 ActiveBlades.Add(blade);
                 UpdateLayout();
-                GetScrollViewer()?.ChangeView(_scrollViewer.ScrollableWidth, null, null);
+
+                if (BladeMode == BladeMode.Normal)
+                {
+                    GetScrollViewer()?.ChangeView(_scrollViewer.ScrollableWidth, null, null);
+                }
+
+                if (BladeMode == BladeMode.Fullscreen)
+                {
+                    for (int x = 1; x < ActiveBlades.Count; x++)
+                    {
```

Comment:
Scroll to the end for ever item? So if there are nine blades, it scrolls eight times? And with the odd delay, that's 4 seconds to wait for it to scroll.